src/style.css

```diff
@@ -0,0 +1,138 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.page {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.page__card {
+  width: 183px;
+}
+
+.card__photo {
+  display: block;
+  margin: 0 auto 40px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+
+  font-family: inherit;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin: 0;
+
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 10px;
+
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+  height: 16px;
+}
+
+.card__reviews-stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 > :nth-child(1),
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.card__reviews-text {
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px 0 auto;
```

Comment:
Will be better to use margin-top / margin-bottom properties for all card elements